artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/xml/XmlDataImporter.java

```diff
@@ -90,10 +91,12 @@ public final class XmlDataImporter extends ActionAbstract {
    private ClientSession session;
    private ClientProducer producer;
 
-   @Option(names = "--host", description = "The host used to import the data. Default: localhost.")
-   public String host = "localhost";
+   @Deprecated(forRemoval = true)
+   @Option(names = "--host", description = "The host used to import the data. Default: null. Deprecated! Use 'url' instead.")
```

Comment:
just use hidden=true, no need to deprecate things in the CLI.

Reply:
I added `hidden = true`, but I left `@Deprecated` so that we will remember to remove it at the next major version.